src/parser/jobs/ast/modules/ArcanaSuggestions.js

```diff
@@ -158,13 +166,24 @@ export default class ArcanaSuggestions extends Module {
 
 						{this.cardLogs.map(artifact => {
 							return <Table.Row key={artifact.timestamp} className={styles.cardActionRow}>
-								<Table.Cell>{this.parser.formatTimestamp(artifact.timestamp)}</Table.Cell>
+
+								<Table.Cell>
+									<a onClick={() =>
+										this.timeline.show((artifact.timestamp-timelineLinkLowerMod) - this.parser.fight.start_time, (artifact.timestamp+timelineLinkUpperMod) - this.parser.fight.start_time)}>
```

Comment:
would be good to get this callback out of the jsx

Reply:
I concur